frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -71,6 +71,7 @@ type CVDToolInstanceManager struct {
 	execContext        ExecContext
 	paths              IMPaths
 	om                 OperationManager
+	uadr               UserArtifactsDirResolver
```

Comment:
Avoid abbreviation in struct members. Most of the time this is going to be used as `m.uadr.SomeFunc()`: The `m` name is fine because the variable is defined close to where it's used, but the `uadr` field is not, I had to search for the struct definition to find the type.

Reply:
agree done.